os_migrate/plugins/module_utils/network.py

```diff
@@ -55,6 +55,36 @@ def serialize_network(sdk_net, net_refs):
 
     return resource
 
+def serialize_security_group(sdk_sec):
+    expected_type = openstack.network.v2.security_group.SecurityGroup
+    if type(sdk_sec) != expected_type:
+        raise exc.UnexpectedResourceType(expected_type, type(sdk_sec))
+
+    resource = {}
+    params = {}
+    info = {}
+    resource['params'] = params
+    resource['info'] = info
+    resource['type'] = 'openstack.network.SecurityGroup'
+
+    params['security_group_rules'] = sdk_sec['security_group_rules']
+    params['tags'] = sorted(sdk_sec['tags'])
+
+    # Decide which attrs are param and which are info
+    set_ser_params_same_name(params, sdk_sec, [
+        'id',
+        'name',
+        'tenant_id',
+        'description',
+        'created_at',
+        'updated_at',
+        'revision_number',
+        'project_id',
+        'location',
+    ])
```

Comment:
Most of these are probably '_info' rather than 'params'. The only params are probably 'name' and 'description'. The rest is probably not needed during import and idempotency checks, so it should got ot '_info'. (I still need to do the change from 'info' to '_info', and global ignoring when comparing resources, as agreed on IRC on Friday.)

Reply:
done